ImportService/lib/import_service-stack.ts

```diff
@@ -0,0 +1,100 @@
+import * as cdk from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import { CfnOutput, RemovalPolicy } from 'aws-cdk-lib';
+
+import {Bucket, BlockPublicAccess, HttpMethods, EventType} from "aws-cdk-lib/aws-s3"
+import { NodejsFunction } from 'aws-cdk-lib/aws-lambda-nodejs';
+import { Runtime } from 'aws-cdk-lib/aws-lambda';
+import { join } from 'node:path';
+import { Deployment, LambdaIntegration, LambdaRestApi, Stage } from 'aws-cdk-lib/aws-apigateway';
+import { CorsHttpMethod } from 'aws-cdk-lib/aws-apigatewayv2';
+import { LambdaDestination } from 'aws-cdk-lib/aws-s3-notifications';
+
+
+export class ImportServiceStack extends cdk.Stack {
+  constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+
+    const bucket = new Bucket(this, 'ImportProductsFileBucket', {
+      blockPublicAccess: BlockPublicAccess.BLOCK_ALL,
+      removalPolicy: RemovalPolicy.DESTROY,
+      autoDeleteObjects: true,
+      cors: [{
+        allowedOrigins: ["*"],
+        allowedHeaders: ["*"],
+        maxAge: 3600,
+        allowedMethods: [HttpMethods.PUT, HttpMethods.POST]
+      }]
+    });
+
+    const destinationBucket = new Bucket(this, 'ProductsFilesDestinationBucket', {
```

Comment:
`uploaded`/`parsed` prefixes should be in the same bucket accordingly to the reqs.

Reply:
Yep, I know. This is why I left this note in the description)

> Note: I decided not to use the same bucket for parsed files just to be safe as this is not a best practice. I'm not focused much on points, so please evaluate this requirement as you see fit.